src/org/usfirst/frc/team467/robot/Drive.java

```diff
@@ -308,19 +311,67 @@ public double getTurnError() {
 	 */
 	public void stop() {
 		//TODO: Stop all motors
-		go(0,0, ControlMode.Disabled);
+		go(0, 0, ControlMode.Disabled);
 		
 	}
 	
 	@Override
 	public void arcadeDrive(double xSpeed, double zRotation, boolean squaredInputs) {
 		super.arcadeDrive(xSpeed, zRotation, squaredInputs);
 		
-		leftFollower1.set(ControlMode.Follower, leftLead.getDeviceID());
-		leftFollower2.set(ControlMode.Follower, leftLead.getDeviceID());
 		
-		rightFollower1.set(ControlMode.Follower, rightLead.getDeviceID());
-		rightFollower2.set(ControlMode.Follower, rightLead.getDeviceID());
+		leftFollower1.follow(leftLead);
+		leftFollower2.follow(leftLead);
+		
+		rightFollower1.follow(rightLead);
+		rightFollower2.follow(rightLead);
+		
+		
+	}
+	
+	public int getLeftSensorVelocity() {
+		return leftLead.getSelectedSensorVelocity(0);
+	}
+	
+	public int getLeftSensorPosition() {
+		return leftLead.getSelectedSensorPosition(0);
 	}
+	
+	public int getRightSensorPosition() {
+		return rightLead.getSelectedSensorPosition(0);
+	}
+	
+	public int getRightSensorVelocity() {
+		return rightLead.getSelectedSensorVelocity(0);
+	}
+	
+	public String getLeftSensorLogData() {
+		return "Vel=" + String.valueOf(getLeftSensorVelocity()) + "Pos=" + String.valueOf(getLeftSensorPosition());
+		
+	}
+	
+	public String getRightSensorLogData() {
+		return "Vel=" + String.valueOf(getRightSensorVelocity()) + "Pos= " + String.valueOf(getRightSensorPosition());
+	}
+	
+	
+	
+	
 
+	public double quadSensorValueLeftPos() {
+		double leftPos = leftLead.getSelectedSensorPosition(0);
+		return leftPos;
+	}
+	public double quadSensorValueLeftVel() {
+		double leftVel = leftLead.getSelectedSensorVelocity(0);
+		return leftVel;
+	}
+	public double quadSensorValueRightPos() {
+		double rightPos = rightLead.getSelectedSensorPosition(0);
+		return rightPos;
+	}
+	public double quadSensorValueRightVel() {
+		double rightVel = rightLead.getSelectedSensorVelocity(0);
+		return rightVel;
+	}
```

Comment:
All of the sensor value changes don't seem to be used anywhere. Where did these changes come from? They should probably be removed.